src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <form 
+      action="http://localhost:8080/" 
+      method="post"
+      data-qa="big">
+      <div class="big-icon">
+        <img 
+          class="pic" 
+          src="images/Search.svg"
+          alt="search-icon"
+          >
+        </img>
+        <input
+          data-qa="keypress"
+          class="bar big-bar"  
```

Comment:
```suggestion
          class="bar"  
```

Reply:
add all styles for big form to bar class and then rewrite of them for small form, cause most of the styles are common for them